SCClassLibrary/Common/Audio/UGen.sc

```diff
@@ -283,6 +283,16 @@ UGen : AbstractFunction {
 		^ModDif.multiNew(this.rate, this, that, mod)
 	}
 
+	// |==| for other AbstractFunctions should compose the |==| operator itself
+	// |==| is not supported in scsynth, so, compose it to == which is supported
```

Comment:
The comment is a bit confusing. I think you can leave it out.

Reply:
The issue is:

```
p = Pseries(0, 1, inf) |==| 2;
p.asCompileString

(Pseries() |==| 2)
```

Because Pbinop represents itself this way, we can't compose `|==|` as `==` -- if Pbinop's operator is `==`, then the compile string would write `(Pseries() == 2)` and that's a different meaning.

*But*, if `aUGen |==| 2` composes `|==|` into the BinaryOpUGen, it fails with "ERROR: Operator '|==|' applied to a UGen is not supported in scsynth." So UGens need to replace `|==|` with `==`.

I can anticipate a future maintainer wondering why patterns and UGens compose differently. I think it's better to explain this.

Or, change Pbinop's compile string so that it writes `Pbinop('==', Pseries(), 2)`. Then the operator being composed can be consistent everywhere.